hack/local_setup.sh

```diff
@@ -0,0 +1,152 @@
+# THE SCRIPT SHOULD BE RUN WHILE IN THE MCM REPOSITRY IN HACK FOLDER & IT ASSUMES THAT IN THE PARENT DIRECTORY OF MCM
+# THE MCM PROVIDER IS ALSO PRESENT
+
+#HOW TO CALL 
+################################################################################################
+
+# bash local_setup.sh --PROJECT <project-name> --SEED <seed-name> --SHOOT <shoot-name>
+
+################################################################################################
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+declare SEED 
+declare SHOOT
+declare PROJECT 
+declare CURRENT_DIR
+declare PROJECT_ROOT
+declare KUBECONFIG_PATH
+declare PROVIDER_PATH
+declare PROVIDER_KUBECONFIG_PATH
+
+while [ $# -gt 0 ]; do
+
+if [[ $1 == *"--"* ]]; then
+     v="${1/--/}"
+     declare $v="$2"
+fi
+
+shift
+done
+
+## to check if something is null and ask to give it again
+# if [ -z "$PROJECT" ]
+# then
+#       echo "enter PROJECT name"
+# 	read PROJECT
+# fi
+
+# if [ -z "$SEED" ]
+# then
+#       echo "enter SEED name"
+# 	read SEED
+# fi
+
+# if [ -z "$SHOOT" ]
+# then
+#       echo "enter SHOOT name"
+# 	read SHOOT
+# fi
```

Comment:
before raising a PR , we should always remove the commented code.
Otherwise you can raise a draft PR

Reply:
oh ok, so I will just remove this code, but there is some code that I want to keep as alternate ways of doing something, if in future this method doesn't work, so should I just leave them, or is there any tag for them like "TODO:"

Reply:
only keep the code which you think will take some time to reinvent, and mark it as `ALTERNATIVE APPROACH`